server/src/main/kotlin/com/kamelia/jellyfish/util/Extensions.kt

```diff
@@ -79,30 +79,30 @@ fun PipelineContext<*, ApplicationCall>.userOrNull(): UserPrincipal? = this.call
 val PipelineContext<*, ApplicationCall>.jwt: Payload
     get() = jwtOrNull() ?: throw ExpiredOrInvalidTokenException()
 
-val PipelineContext<*, ApplicationCall>.authenticatedUser: User
-    get() = userOrNull()?.user ?: throw ExpiredOrInvalidTokenException()
+val PipelineContext<*, ApplicationCall>.authenticatedUser: UserState?
+    get() = userOrNull()?.state
 
-val PipelineContext<*, ApplicationCall>.accessToken: String
-    get() = userOrNull()?.token ?: throw ExpiredOrInvalidTokenException()
+val PipelineContext<*, ApplicationCall>.accessToken: String?
+    get() = userOrNull()?.accessToken
```

Comment:
Because if I try to get the current authenticated user and nobody is logged in, the call ends with a 401 even though I would want to continue, knowing I'm dealing with a guest in this context.

Reply:
fair enough